drivers/gpu/drm/vc4/vc4_crtc.c

```diff
@@ -343,6 +343,10 @@ static void vc4_crtc_config_pv(struct drm_crtc *crtc)
 	bool is_dsi1 = vc4_encoder->type == VC4_ENCODER_TYPE_DSI1;
 	u32 format = is_dsi1 ? PV_CONTROL_FORMAT_DSIV_24 : PV_CONTROL_FORMAT_24;
 	u8 ppc = pv_data->pixels_per_clock;
+	bool vec_interlaced = (vc4_encoder->type == VC4_ENCODER_TYPE_VEC &&
+			       interlace);
+	bool ntsc = (vec_interlaced && mode->htotal == 858);
```

Comment:
I now realize that this may even be confusing, as this actually detects 525/625-line modes, not NTSC per se (i.e. PAL-M is "NTSC" in this sense and that's intended), so renaming these variables is probably the right thing to do.